### DISCUSSION TITLE
F3 for Play / pause should work in multiplayer lobbies

### DISCUSSION THREAD

**Discussion by Sinamer** [ppy/osu#15131] [Ideas] [answered]
Sometimes the break time in multi lobbies is quite long and the music looping gets annoying, the only way to stop it would be to open the "now playing" menu and click the button. Since F3 is already a game-wide shortcut for Play / pause it would be nice if it also worked in multiplayer lobby screen.

**Comment by bdach** [comment]
Not sure. What if the track finally changes due to someone changing the map? Should playback resume or not?

Muting tracks only via the volume meter / settings sidebar is a possible temporary workaround. Effects will still play if you touch only the track volume and not the effect volume.

**Reply by Sinamer** [reply]
Yes, I would expect the playback to resume once the beatmap changes, that's how it works in osu!stable. There's a dedicated Play / pause button on the screen in stable multiplayer so I assume pausing the music is quite common for everyone who played multiplayer before.

**Comment by peppy** [comment]
this is definitely a duplicate issue, i swear we've discussed this before.

**Reply by bdach** [reply]
now that you mention it, i also seem to recall that but can't find it... https://github.com/ppy/osu/discussions/13173 maybe?

**Reply by peppy** [reply]
That's the one!

**Reply by Sinamer** [reply]
As much as that discussion is about a similar issue, it is different from what I'm asking for. It's weird when a game-wide hotkey suddenly stops working when you join a lobby. I'm simply asking for it to work where it's reasonable for it to work, since it's intuitive to use keyboard for playback control.

**Comment by peppy** [comment, marked as answer]
Should be resolved via https://github.com/ppy/osu/pull/16977.
